app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/setpassword/SetPasswordScreen.kt

```diff
@@ -170,7 +170,7 @@ private fun SetPasswordScreenContent(
         Spacer(modifier = Modifier.height(16.dp))
 
         BitwardenTextField(
-            label = stringResource(id = R.string.master_password_hint),
+            label = stringResource(id = R.string.master_password_hint_optional),
```

Comment:
Why change the name?

Reply:
Hmmm, I see why you did it now.

@SaintPatrck is this going to confuse Crowdin?

Reply:
I can make the new one "m_p_h_not_optional" haha

Reply:
not_specified *

Reply:
I think that would work

Reply:
> Hmmm, I see why you did it now.
> 
> @SaintPatrck is this going to confuse Crowdin?

I see you reverted renaming that resource so it's moot now, but for future reference, renaming will result in loss of existing translations for the resource being renamed.

Reply:
OK, thanks @SaintPatrck